win/WIN_Utils.h

```diff
@@ -8,7 +8,7 @@ namespace win
 {
 	namespace utils
 	{
-		bool filterNumerical(const char c);
+		bool filterNumerical(char c);
 
 		int findOutcode(gfx::Rectangle rect, int x, int y);
```

Comment:
Should be part of gfx.
**MUST**

Reply:
Done, outcode stuff is now in gfx. Win utils contains only filterNumerical and getMouseState now